src/server/shared/SharedDefines.h

```diff
@@ -1650,7 +1650,7 @@ enum GOState : uint8
 {
     GO_STATE_ACTIVE             = 0,                        // show in world as used and not reset (closed door open)
     GO_STATE_READY              = 1,                        // show in world as ready (closed door close)
-    GO_STATE_ACTIVE_ALTERNATIVE = 2                         // show in world as used in alt way and not reset (closed door open by cannon fire)
+    GO_STATE_DESTROYED          = 2                         // show in world as used in alt way and not reset (closed door open by cannon fire)
```

Comment:
Since this is essentially all that is being changed here in this PR, I recommend using a precise description in the comment line, to avoid confusion or misinterpretation.
Preliminary suggestion:
```suggestion
    GO_STATE_DESTROYED          = 2                         // show the object in-game as already used and not yet reset (e.g. door opened by a cannon blast)
```
(of course, this may be a tad much, so feel free to suggest a shorter version)